### PR TITLE
Update uploads pacakge

### PR DESCRIPTION
**Pull request by aspirisen**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "sourceMap": true,
         "strict": true,
         "target": "es5",
-
+        "skipLibCheck": true,
```
Comment (e1himself) [inline tsconfig.json:19]: Why is this needed? :thinking: Doesn't it make the type-checking weaker?

**Review by e1himself — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (aspirisen) [reply tsconfig.json:19]: There was issues in library types in case when there was no extraneous dependencies in pnpm

Comment (e1himself) [reply tsconfig.json:19]: I don't understand. Can you please elaborate more on it?
